cnxcommon/urlslug.py

```diff
@@ -58,7 +58,8 @@ def generate_slug(book_title, *other_titles):
 
 @utils.ensure_unicode
 def remove_html_tags(title):
-    return re.sub(r"<.*?>", "", title)
+    tmp_title = re.sub(r'<span class="os-part-text">([^<]+)</span>', "", title)
```

Comment:
regex is very very brittle when used on xml. In this case, modifying the element's attributes in any way will break this. I suppose we use the same method in `get_os_number` as well, and it's been in production this way for quite a while, though... Maybe not such a big deal.
If we are confident the recipe won't change, then it's fine by me.